<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frmFile">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>File Name:</string>
           </property>
           <property name="buddy">
            <cstring>txtCSVFileName</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QLineEdit" name="txtCSVFileName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="btnSaveCSVFile">
           <property name="text">
            <string>Select File Name ...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>27</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QCheckBox" name="chkWriteHeader">
           <property name="text">
            <string>Output Header Column Names</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="toolTip">
            <string>The character used between fields.</string>
           </property>
           <property name="text">
            <string>Delimiter:</string>
           </property>
           <property name="buddy">
            <cstring>cmbDelimiter</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="cmbDelimiter">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>,</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>/</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>#</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>|</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enter a character</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="4">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="toolTip">
            <string>The character used to quote text fields</string>
           </property>
           <property name="text">
            <string>Quote Character:</string>
           </property>
           <property name="buddy">
            <cstring>cmbQuoteChar</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="3" colspan="2">
          <widget class="QComboBox" name="cmbQuoteChar">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>'</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>&quot;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enter a Character</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="5">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_8">
           <property name="toolTip">
            <string>Defines when quotes will be used on text fields.</string>
           </property>
           <property name="text">
            <string>Use Quotes:</string>
           </property>
           <property name="buddy">
            <cstring>cmbUseQuotes</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="3">
          <widget class="QComboBox" name="cmbUseQuotes">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Minimal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Non-Numeric</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="5" colspan="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>104</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="toolTip">
            <string>True will double the quote character when it appears inside a quoted string.  False will use the escape character.</string>
           </property>
           <property name="text">
            <string>Doube Quote:</string>
           </property>
           <property name="buddy">
            <cstring>cmbDoubleQuote</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="2" colspan="3">
          <widget class="QComboBox" name="cmbDoubleQuote">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>TRUE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FALSE</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="5" colspan="2">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>104</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QLabel" name="label_10">
           <property name="toolTip">
            <string>The character used between fields.</string>
           </property>
           <property name="text">
            <string>Escape Character:</string>
           </property>
           <property name="buddy">
            <cstring>cmbEscapeCharacter</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="3" colspan="2">
          <widget class="QComboBox" name="cmbEscapeCharacter">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>/</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>#</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>|</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Enter a character</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="5" colspan="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>104</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0" colspan="3">
          <widget class="QLabel" name="label_11">
           <property name="toolTip">
            <string>The character used between fields.</string>
           </property>
           <property name="text">
            <string>Line Terminator:</string>
           </property>
           <property name="buddy">
            <cstring>cmbLineTerminator</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="3" colspan="2">
          <widget class="QComboBox" name="cmbLineTerminator">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>\r\n</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>\r</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>\n</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="5" colspan="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>104</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
